pages/ChatList/index.tsx

```diff
@@ -3,30 +3,29 @@ import React from 'react';
 import { ScrollView } from 'react-native';
 import { RouteComponentProps } from 'react-router-native';
 import ChatItem from '../../components/Chat/ChatItem';
-import ChatTitleBar from '../../components/Chat/ChatTitleBar';
+import AppLogo from '../../components/Chat/AppLogo';
 import { connect } from "react-redux";
-import { setThemeAction } from '../../store/reducers/config';
 import { Dispatch } from 'redux';
-import { ApplicationConfig, AppTheme } from '../../config/DefaultConfig';
 
 interface Props extends RouteComponentProps {
   dispatch: Dispatch
 }
 
 const ChatList: React.FunctionComponent<Props> = ({
-  dispatch,
   history
 }: Props) => {
 
   const goToChatDetails = () => {
     history.push('/chat')
   }
 
-  const updateTheme = (theme: AppTheme) => dispatch(setThemeAction(theme))
+  const goToChatSetting = () => {
+    history.push('/setting')
+  }
 
   return (
     <>
-      <ChatTitleBar updateTheme={updateTheme} />
+      <AppLogo userImageSource={{ uri: "https://recraftrelic.com/images/Recraft_relic_web_logo_icon.png" }} onSettingPress={goToChatSetting}/>
```

Comment:
* Move the AppLogo image to `constants`.
* change `userImageSource` to `appLogoSource`

Reply:
I have moved the AppLogo image source to constants & also renamed the imagesource prop